test_utils/utils.go

```diff
@@ -65,7 +138,7 @@ func RunIPAMPlugin(netconf, command, args string) (types.Result, int) {
 	return result, exitCode
 }
 
-func CreateContainer(netconf string) (container_id, netnspath string, session *gexec.Session, contVeth netlink.Link, contAddr []netlink.Addr, contRoutes []netlink.Route, err error) {
+func CreateContainer(netconf string, name string) (container_id, netnspath string, session *gexec.Session, contVeth netlink.Link, contAddr []netlink.Addr, contRoutes []netlink.Route, err error) {
```

Comment:
should name be k8sName or something? Not immediately obvious it's a k8s only thing